interp/cfg.go

```diff
@@ -1311,6 +1311,10 @@ func (interp *Interpreter) cfg(root *node, importPath string) ([]*node, error) {
 					break
 				}
 			}
+			if len(n.child) > sc.def.typ.numOut() {
+				err = n.cfgErrorf("too many arguments to return")
+				break
+			}
```

Comment:
Is there any particular reason that this is done here, instead of above 1304? Is it better here ?

Reply:
No, this test could indeed be done before the previous one.